packages/teleport/src/Databases/AddDatabase/AddDatabase.tsx

```diff
@@ -30,7 +30,7 @@ export default function AddDatabase({ user, version, onClose }: Props) {
       onClose={onClose}
       open={true}
     >
-      <DialogTitle mb={4} mr="auto">
+      <DialogTitle mb="4" mr="auto">
```

Comment:
i think curly brackets more accurate b/c its a number, it refers to the indexes of this array: https://github.com/gravitational/webapps/blob/master/packages/design/src/theme/theme.js#L22, the values are in pixels, and keep in mind, anytime for spacings we want to use these values (multiples of 8) for consistency